src/lib/src/services/validators/validators.service.mock.ts

```diff
@@ -73,6 +73,16 @@ export class TsValidatorsServiceMock {
     });
   }
 
+  lowercase = () => {
+    return jest.fn().mockReturnValue(this.isValid ? null : {
+      lowercase: {
+        valid: false,
+        contains: 2,
+        actual: 6,
```

Comment:
a) `actual` should be a string of letters since it is the fake version of what might be typed into the input. (eg `actual: 'FOo'`)
b) As mentioned in another comment below: the key `contains` should be the validator name. So in this case it would be `lowercase: 2,`.